app/uk/gov/hmrc/cataloguefrontend/healthindicators/HealthIndicatorsConnector.scala

```diff
@@ -30,88 +30,88 @@ class HealthIndicatorsConnector @Inject()(
 )(implicit val ec: ExecutionContext) {
   import HttpReads.Implicits._
 
-  private implicit val repositoryRatingReads: Reads[RepositoryRating] = RepositoryRating.reads
+  private implicit val indicatorReads: Reads[Indicator] = Indicator.reads
 
   private val healthIndicatorsBaseUrl: String = servicesConfig.baseUrl("health-indicators")
 
-  def getRepositoryRating(repoName: String)(implicit hc: HeaderCarrier): Future[Option[RepositoryRating]] = {
+  def getIndicator(repoName: String)(implicit hc: HeaderCarrier): Future[Option[Indicator]] = {
     val url = s"$healthIndicatorsBaseUrl/health-indicators/repositories/$repoName"
     http
-      .GET[Option[RepositoryRating]](url)
+      .GET[Option[Indicator]](url)
   }
 
-  def getAllRepositoryRatings(repoType: RepoType)(implicit hc: HeaderCarrier): Future[Seq[RepositoryRating]] = {
+  def getAllIndicators(repoType: RepoType)(implicit hc: HeaderCarrier): Future[Seq[Indicator]] = {
     // AllTypes is represented on the backend by sending no RepoType parameter
     val repoTypeQueryP = if(repoType == RepoType.AllTypes) "" else "&repoType=" + repoType.asString
     val url = s"$healthIndicatorsBaseUrl/health-indicators/repositories/?sort=desc" ++ repoTypeQueryP
```

Comment:
It is unrelated to your change but I think we should not create a URL like this by string interpolation. The `GET` method accepts a list of query params. We can pass the query params as the argument to that parameter.

Reply:
Can you make this change please?